.github/workflows/build.yml

```
@@ -34,6 +34,11 @@ jobs:
           sudo cp $GITHUB_WORKSPACE/_resources/bin/plantuml /usr/local/bin
           sudo chmod +x /usr/local/bin/plantuml
 
+      - name: Install Node.js 14.x
+        uses: percona-platform/setup-node@v2.1.2
```

```suggestion
        uses: percona-platform/setup-node@v2
```
We upgraded the script, it should now be just v2.